$NetBSD: patch-ab,v 1.18 2007/09/18 10:44:57 wiz Exp $

http://bugzilla.gnome.org/show_bug.cgi?id=476342

From upstream SVN:
   Revision 18823 - (view) (annotate) - [select for diffs]
   Modified Fri Sep 14 02:11:01 2007 UTC (4 days, 6 hours ago) by matthiasc
   File length: 13336 byte(s)
   Diff to previous 17898

 Turn off icon cache validation by default

--- gtk/gtkiconcache.c.orig	2007-09-13 20:35:41.000000000 +0000
+++ gtk/gtkiconcache.c
@@ -127,6 +127,9 @@ _gtk_icon_cache_new_for_path (const gcha
   info.n_directories = 0;
   info.flags = CHECK_OFFSETS|CHECK_STRINGS;
 
+#ifdef G_ENABLE_DEBUG
+  if (gtk_debug_flags & GTK_DEBUG_ICONTHEME)
+    {
   if (!_gtk_icon_cache_validate (&info))
     {
       g_mapped_file_free (map);
@@ -134,6 +137,8 @@ _gtk_icon_cache_new_for_path (const gcha
 
       goto done;
     }
+    }
+#endif
   
   GTK_NOTE (ICONTHEME, g_print ("found cache for %s\n", path));
 
